### ISSUE TITLE
Claymore Suddenly stopped working.  

### ISSUE THREAD

**Issue by Techmat** [nanopool/Claymore-Dual-Miner#252]
Hi, I was mining with 4 AMD card about six months with nanopool and suddenly my miner stopped working. I've tried to download the latest version, tried with old version nothing is working, it just do nothing after open. See the picture for example. 

Does anyone have any idea what can be wrong? Thanks in advanced. 
<img width="855" alt="screen shot 2018-05-18 at 23 08 20" src="https://user-images.githubusercontent.com/22291283/40260148-0e1083d2-5af1-11e8-9d30-fbed2cd6b2ca.png">
<img width="278" alt="screen shot 2018-05-18 at 23 09 03" src="https://user-images.githubusercontent.com/22291283/40260149-0e320282-5af1-11e8-969d-373bbf894032.png">

**Comment by intek2016** [comment]
yes i have same problem too. any one can help ?

**Comment by manta828** [comment]
Hello, i have the same problem.I open the mining program, it show's the video cards and then it stop's. Can you please help me ?

**Comment by manta828** [comment]
I reinstaled windows 3times and still no change. After i open the program, it starts, shows the video cards and then it stops or my monitor goes black. Did you also have this problem ?

**Comment by intek2016** [comment]
yes same but my monitor not blank like u. may be u can ask to this forum by seding the log file 

https://bitcointalk.org/index.php?topic=1433925.23960

**Comment by intek2016** [comment]
or do you update windows ? @manta828

**Comment by manta828** [comment]
No, i installed a fresh new copy of windows. Now im installing an older version of windows 10 to see if this works

**Comment by manta828** [comment]
After i install the atikmdag patcher 1.4.6 it gives the black screen when i open the mining program. Does anyone had this problem ? My graphic cards are rx 570itx pulse and im mining eth

**Comment by Techmat** [comment]
Hi, I've installed new windows and it seems to be working normal like before, I'm testing it that if few hours later or few days later it happens again or not.

**Comment by manta828** [comment]
After i installed windows for the 5th time, i installed the latest amd driver and i was getting 15-17mh per video card. After that i installed an older version of amd 17.8 and it works great. I have 26-29mh. Hope this will help others

**Comment by IvanMitov** [comment]
Hi, I`ve reinstalled windows 3 times, program start but after 1-2 hours stop and problem is the same. I tried with different drivers - nothing... I know the problem is in windows, but I don`t know what exactly is the problem. Can anyone help? - 6xNvidia 1060 6GB, 4GB RAM. I have two rigs and after update of that shitty windows problems appeared...

**Comment by jaeckelbmw** [comment]
Has anyone found a solution to this issue yet? Only thing I can think of next would be to take it all apart and see if it is hardware related.

**Comment by manta828** [comment]
Try and reinstall windows. Also check your cables, my power supply melted
some of the wires and had to replace them.

**Comment by jaeckelbmw** [comment]
Taking it apart and looking everything over is the only thing I haven't done BUT it does run FinMiner, the  "qiuck start" option from Nanopool.

**Comment by IvanMitov** [comment]
For me the problem was drivers of built-in video card. Just uninstalled drivers with DDU and everything started to work normally.

**Comment by jaeckelbmw** [comment]
So do the DDU on the Mobo video card? Seems random that that damn thing would be causing the problem, but i'll give it a try. And since I am using that Mobo video for the monitor when I have that connected, shouldn't it fix itself?
